src/libraries/System.Net.Http/tests/StressTests/HttpStress/windows.Dockerfile

```diff
@@ -14,6 +14,11 @@ COPY . .
 ARG CONFIGURATION=Release
 RUN dotnet build -c $env:CONFIGURATION
 
+# Enable dump collection
+ENV COMPlus_DbgEnableMiniDump=1
```

Comment:
Is there any side effect from enabling this always?

Reply:
AFAIK nope, this will generate dump only if the process crashes. It shouldn't have any effect on the running process.